test/test_jsoo_react.re

```diff
@@ -1058,6 +1072,7 @@ let externals =
     "basic" >:: testExternals,
     "children" >:: testExternalChildren,
     "non-function" >:: testExternalNonFunction,
+    "optioal-arg" >:: testExternalOptionalArg,
```

Comment:
```suggestion
    "optional-arg" >:: testExternalOptionalArg,
```

Reply:
Fixed in https://github.com/ml-in-barcelona/jsoo-react/commit/347b4ef114dd8d2a789e0248005e4461b1c9e9e7